resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/submit/v2/ClientV2Suite.scala

```diff
@@ -205,8 +216,8 @@ class ClientV2Suite extends SparkFunSuite with BeforeAndAfter {
 
   // Tests with local dependencies with the mounted dependency manager.
   test("Uploading local dependencies should create Kubernetes secrets and config map") {
-    val initContainerConfigMap = getInitContainerConfigMap()
-    val initContainerSecret = getInitContainerSecret()
```

Comment:
after this change is `initContainerSecret`'s type `ConfigMap` or a `() -> ConfigMap` function that creates it?

What's Spark style?  cc @tnachen